tests/e2e/specFiles/Transactions/send.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { name } from 'dayjs/locale/*';
+import { expect } from 'detox';
+
+import { isBeta, createRandomNote, WALLETS_WITH_COINS, DATA_FOR_TRANSACTIONS } from '../../helpers';
+import app from '../../pageObjects';
+
+describe('Transactions', () => {
+  beforeEach(async () => {
+    isBeta() && (await app.onboarding.betaVersionScreen.close());
+    await app.developerRoom.tapOnSkipOnboardingButton();
+    await app.navigationBar.changeTab('wallets');
+  });
+
+  describe('Send', () => {
+    describe('@android @ios @regression', () => {
+      it('should be possible to send coins from 3-Key Vault wallet (Secure Transaction) to 3-Key Vault wallet', async () => {
+        const transactionNote = createRandomNote();
+        await app.wallets.importExistingWallet({
+          type: '3-Key Vault',
+          name: '3-Key wallet',
+          fastPublicKey: WALLETS_WITH_COINS['3-Keys Vault'].FAST_KEY,
+          cancelPublicKey: WALLETS_WITH_COINS['3-Keys Vault'].CANCEL_KEY,
+          seedPhrase: WALLETS_WITH_COINS['3-Keys Vault'].SEED_PHRASE,
+        });
+        await app.wallets.dashboardScreen.tapOnSendButton();
+        await app.transactionsSend.sendCoinsMainScreen.typeCoinsAmountToSend(DATA_FOR_TRANSACTIONS.AMOUNT_TO_SEND);
+        await app.transactionsSend.sendCoinsMainScreen.typeWalletAddress(DATA_FOR_TRANSACTIONS.WALLET_ADDRESS);
+        await app.transactionsSend.sendCoinsMainScreen.typeNote(transactionNote);
+        await app.transactionsSend.sendCoinsMainScreen.chooseTransactionType('Secure');
+        await app.transactionsSend.sendCoinsMainScreen.tapNextButtton();
+        await app.transactionsSend.sendCoinsConfirmationScreen.tapSendButton();
+        await app.transactionsSend.sendCoinsPasswordScreen.typePassword(
+          DATA_FOR_TRANSACTIONS.CONFIRM_TRANSACTION_PASSWORD,
+        );
+        await app.transactionsSend.sendCoinsPasswordScreen.tapConfirmPasswordButton();
+        await app.transactionsSend.successScreen.close();
+        await app.wallets.dashboardScreen.scrollToTheTransactionWithNote(transactionNote);
+        await expect(app.wallets.dashboardScreen.getTransactionElement(transactionNote)).toBeVisible();
```

Comment:
Actually, `app.wallets.dashboardScreen.scrollToTheTransactionWithNote` already does this assertion

Reply:
Good point!